apps/tng-dictionary/src/components/TermIndex/TermIndex.tsx

```diff
@@ -1,4 +1,3 @@
-
```

Comment:
Should this file update be in this PR?

Reply:
No- looks like formatting diffs only, too. I'll reset this.

Reply:
done.